source/clear-linux/guides/maintenance/download-image.rst

```diff
@@ -0,0 +1,192 @@
+.. _download-image:
+
+Download, verify, and uncompress a Clear Linux image
+####################################################
+
+.. _types-of-cl-images:
+
+Types of Clear Linux images
+===========================
+
+|CL| offers many types of images for different platforms and environments.
+
+The naming convention of a |CL| image filename is: 
+
+  .. code-block:: console
+   
+    clear-<version number>-<image type>.<compression type>
+
+The `<version number>` field specifies the version number.
+
+The `<image type>` field specifies the type of image and its corresponding file format.  The current list of images available are:
+
+* `aws.img`: Image for Amazon AWS.
+* `azure.vhd`: Image for Microsoft Azure.
+* `azure-docker.vhd`: An Azure image with the Docker installed.
+* `azure-machine-learning.vhd`: An Azure image with the Machine Learning bundle installed.
+* `cloud.img`: Image intended for cloud deployment (openstack, etc.).
+* `containers.img`: An optimized image utilised by Clear Containers runtime.
+* `gce.img`: Image for Google Compute Engine.  
+* `hyperv.vhdx`: Image for Microsoft HyperV.
+* `hyperv-mini.vhdx`: A stripped down HyperV image with fewer bundles.
+* `installer.img`: An interactive installer image for installing Clear Linux.
+* `installer.iso`: An ISO of interactive installer image for installing Clear Linux.
+* `kvm.img`: Image intended for booting in a simple VM with start_qemu.sh.
+* `live.img`: A live boot image of Clear Linux.
+* `safety-enhanced-linux.img`: 
+* `vmware.vmdk`: Image for VMware.
+
+The `<compression type>` field specifies the compression type. Two types of 
+compressions are used, namely GNU zip (`.gz`) and XZ (`.xz`).
+
+The latest, validated, and official releases are found in the `image`_ repository.
+Older official releases and unofficial - and not fully validated - releases are 
```

Comment:
There is no way to tell an official release from an unofficial release as far as I can see.  I would change this to "Older releases are found in the releases directory.", removing mention of official and unofficial altogether.